ros_bridge_interface/src/ros_bridge_interface.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+import rospy
+from std_msgs.msg import String
+import json
+import time
+
+
+class RosBridgeInterface:
+    def __init__(self):
+        rospy.init_node('ros_bridge_interface_server', anonymous=True)
+        self.interface_action_sub = rospy.Subscriber("ui/robot/call_action", String, self.interface_action_callback)
+        self.set_screen_pub = rospy.Publisher("ui/app/set_screen", String, queue_size=10)
```

Comment:
Can we change the interface node published topic to "nightingale_ui/robot_status"? I think this makes more sense since the interface is relaying changes in the robot status for the tablet to reach to by changing the screen.